### DISCUSSION TITLE
neovim --head install version

### DISCUSSION THREAD

**Discussion by Trid-collab** [Homebrew/discussions#3433] [Linux]
Why is `brew install neovim --HEAD` installing the older versions of the nightly suddenly 

`NVIM v0.8.0-dev-nightly-184-g1c9f487f0
Build type: Release
LuaJIT 2.1.0-beta3`

Earlier few days back it was  `NVIM v0.8.0-dev+601-g540d1af90`

Am I missing out anything

**Comment by carlocab** [comment]
> Why is `brew install neovim --HEAD` installing the older versions of the nightly suddenly 

It's not.

> ```
> NVIM v0.8.0-dev-nightly-184-g1c9f487f0
> Build type: Release
> LuaJIT 2.1.0-beta3
> ```

This is built from https://github.com/neovim/neovim/commit/1c9f487f0edc3e6a10a53dc83a3d779980dc3a26, which is, at the time of writing, under an hour old.

> Earlier few days back it was  `NVIM v0.8.0-dev+601-g540d1af90`

This is built from https://github.com/neovim/neovim/commit/540d1af90e2276a8bd3b5edb1a63c8225a1aa423, which is about three days old.

**Reply by Trid-collab** [reply]
I have downloaded the nvim nightly from git at the same time on another machine it shows 
` NVIM v0.8.0-dev-nightly-12-g977790746`
what is the difference between this version and the one installed with brew

**Reply by gromgit** [reply]
That's a question only upstream can answer, i.e. https://github.com/neovim/neovim.

**Reply by carlocab** [reply]
The nightly you are referring to is built from https://github.com/neovim/neovim/commit/9777907467b29e890556db287b6a9995c0024896.

This should allow you to compare the differences in the code directly:

https://github.com/neovim/neovim/compare/977790746..1c9f487f0
